src/util/Constants.js

```diff
@@ -1510,6 +1514,15 @@ exports.SortOrderTypes = createEnum([null, 'LATEST_ACTIVITY', 'CREATION_DATE']);
  */
 exports.ForumLayoutTypes = createEnum(['NOT_SET', 'LIST_VIEW', 'GALLERY_VIEW']);
 
+/**
+ * The animation type of the voice channel effect
+ * * PREMIUM
+ * * BASIC
+ * @typedef {string} AnimationType
+ * @see {@link https://discord.com/developers/docs/topics/gateway-events#voice-channel-effect-send-animation-type}
```

Comment:
```suggestion
 * @see {@link https://discord.com/developers/docs/topics/gateway-events#voice-channel-effect-send-animation-types}
```

See: https://github.com/discord/discord-api-docs/pull/6025/files#diff-088ff157a6538362b78b58c1102d85a124842edbe5d558f10c42d2d66bcf0d13